lib/validation/dpValidateMultiselectDirective.js

```diff
@@ -1,4 +1,4 @@
-import { validateMultiselect } from '../../index'
+import validateMultiselect from '../../lib/validation/utils/validateMultiselect'
```

Comment:
This looks unnecessarily complicated. Can't we do
`import { validateMultiselect } from './index.js'`
or
`import validateMultiselect from './utils/validateMultiselect'`?

Reply:
I admit I had to giggle a bit ;)